KinoPub/components/VideoDescriptionPanel.brs

```diff
@@ -43,14 +43,22 @@ end sub
 
 sub itemReceived()
     print "VideoDescriptionPanel:itemReceived"
+    
+    filesCount = m.readItemTask.content.item.videos[0].files.Count()
```

Comment:
This doesn't seem to affect anything, why do you need it? The only code which uses it is under 
if false